src/index.html

```diff
@@ -1,12 +1,297 @@
+              </div>
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+            </div>
+            <p class="product-card__review">Отзывов: 5</p>
+          </div>
+          <div class="product-card__price-row">
+            <p class="product-card__price-tag">Цена:</p>
+            <p class="product-card__price">69 999 грн</p>
+          </div>
+          <a
+            href="/"
+            class="product-card__button"
+          >
+            КУПИТЬ
+          </a>
+        </div>
+        <div class="product-card">
+          <div class="product-card__image">
+          </div>
+          <p class="product-card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+          <p class="product-card__code">Код товара: 195434</p>
+          <div class="product-card__rating">
+            <div class="product-card__stars product-card__stars--4">
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+            </div>
+            <p class="product-card__review">Отзывов: 5</p>
+          </div>
+          <div class="product-card__price-row">
+            <p class="product-card__price-tag">Цена:</p>
+            <p class="product-card__price">69 999 грн</p>
+          </div>
+          <a
+            href="/"
+            class="product-card__button"
+          >
+            КУПИТЬ
+          </a>
+        </div>
+        <div class="product-card">
+          <div class="product-card__image">
+          </div>
+          <p class="product-card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+          <p class="product-card__code">Код товара: 195434</p>
+          <div class="product-card__rating">
+            <div class="product-card__stars product-card__stars--4">
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+            </div>
+            <p class="product-card__review">Отзывов: 5</p>
+          </div>
+          <div class="product-card__price-row">
+            <p class="product-card__price-tag">Цена:</p>
+            <p class="product-card__price">69 999 грн</p>
+          </div>
+          <a
+            href="/"
+            class="product-card__button"
+          >
+            КУПИТЬ
```

Comment:
make text uppercase in css, there should be usual text

Reply:
+